scripts/utils/divviRewardsExcludedReferrers.ts

```diff
@@ -55,10 +55,7 @@ export async function getDivviRewardsExcludedReferrers(): Promise<
     }
   >
 > {
-  // TODO: The fetch to get OFAC SDN addresses is disabled for now because it
-  // intermittently fails with 403.
-  const ofacSdnAddresses: { referrerId: Address; shouldWarn: boolean }[] = []
-  // const ofacSdnAddresses = await getOfacSdnAddresses()
+  const ofacSdnAddresses = await getOfacSdnAddresses()
```

Comment:
could just cache this in a global var here and avoid all the passing around of args.